tests/octoberSky/input.txt

```diff
@@ -3755,7 +3755,7 @@
 177182228725,1,15.3442,-88.867,106.859,10.2732,4.61587,14.3981,45.1412,-75.311,87.11,-22.4419,-22.4419,-1.20935,-0.875416,0.98839,-9.5575,-0.461751,0.65549,258,36992,22.039,22.039,54.719,5,500,500,500,1000,1000,1000,6,101595,1023,0.134622,-0.0036132,0.0119134,11.9002,-9.557,-0.370732,0.715535,0.157352,-0.00815746,0.00695359,1,0
 177215570837,1,-132.644,-89.9802,-103.778,543.586,nan,561.58,45.1412,-75.311,87.11,-22.608,-22.608,-1.20681,-0.88337,0.967496,-9.4485,0.106047,-0.127815,258,36992,22.039,22.039,54.719,5,500,500,500,1000,1000,1000,6,101597,1023,0.199299,0.00588606,-0.00317236,11.9073,-9.45953,0.158669,-0.115503,0.196511,0.0062032,-0.00244648,1,0
 177248961540,1,-132.644,-89.9802,-103.778,543.586,nan,561.58,45.1412,-75.311,87.11,-22.608,-22.608,-1.20681,-0.88337,0.967496,-9.4485,0.106047,-0.127815,258,36992,22.039,22.039,54.719,5,500,500,500,1000,1000,1000,6,101597,1023,0.199299,0.00588606,-0.00317236,11.9073,-9.45953,0.158669,-0.115503,0.196511,0.0062032,-0.00244648,1,0
-177282256244,1,-114.386,-nan,-121.885,610.21,nan,663.584,45.1412,-75.311,87.11,-22.608,-22.608,-1.20858,-0.883102,0.971221,-9.42784,0.12852,-0.169869,258,36992,22.039,22.039,54.719,5,500,500,500,1000,1000,1000,6,101597,1023,0.163795,0.00500334,0.00303238,11.907,-9.44163,0.202418,-0.200985,0.142585,0.00522227,0.00494883,1,0

```

Comment:
In the original parseFloat, a negative NaN, would return a positive NaN. I changed this so the test wouldn't fail, it has no other practical implications.